hack/cloud-image-uploader/go.mod

```diff
@@ -1,20 +1,22 @@
 module github.com/siderolabs/cloud-image-uploader
 
-go 1.20
+go 1.21.0
+
+toolchain go1.21.1
 
 require (
 	github.com/Azure/azure-sdk-for-go v68.0.0+incompatible
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.7.0
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v5 v5.1.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/storage/armstorage v1.3.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.8.0-beta.1
```

Comment:
beta? i don;t really care though

Reply:
yes, it gets pulled in as a dependency of dependency :shrug: